### ISSUE TITLE
Taxonomy tagging

### ISSUE THREAD

**Issue by informatics-dev** [NaturalHistoryMuseum/scratchpads2#4902]
- Original ID: 3766
 - Added by **Mark Pajak**
 - Scratchpads URL: http://bns.myspecies.info/content/abbot%E2%80%99s-pool

### Description:
I am trying to set up tagging, but i am not sure it is working properly. I can see that it searches for species and adds the species to a chosen field. My problem is what happens next, i.e. The text gets added to a field but i had hoped this would be a ling to the species taxon page - should this be the case? If so what steps do i need to set this up. 

To recap - when a block of text includes a listed taxon, i would like an automatic link to the page for that taxon, and if possible a reverse link to the node from the taxon page.

I have tried many different field combos but cant get this to work. any help would be very much appreciated - we aim to go live in january.

### Attachments:
 - https://github.com/NaturalHistoryMuseum/scratchpads2/blob/issue-attachments/677/image.jpg
